web/src/context/AtlasProvider.tsx

```
@@ -0,0 +1,275 @@
+    refetchUser();
+  }, [isVerified, refetchUser]);
+
+  // remove old user's data on address change
+  useEffect(() => {
+    queryClient.removeQueries({ queryKey: ["UserSettings"] });
+  }, [address, queryClient]);
+
+  // this would change based on the fields we have and what defines a user to be existing
+  const userExists = useMemo(() => {
+    if (!user) return false;
+    return user.email ? true : false;
```

Biome

[error] Unnecessary use of boolean literals in conditional expression.

Simplify your code by directly assigning the result without using a ternary operator.
If your goal is negation, you may use the logical NOT (!) or double NOT (!!) operator for clearer and concise code.
 Check for more details about NOT operator.
Unsafe fix: Remove the conditional expression with


(lint/complexity/noUselessTernary)